tools/editor/property_editor.cpp

```diff
@@ -221,6 +221,21 @@ void CustomPropertyEditor::_menu_option(int p_which) {
 						EditorNode::get_singleton()->get_scene_tree_dock()->open_script_dialog(owner->cast_to<Node>());
 
 				} break;
+				case OBJ_MENU_SHOW_IN_FILE_SYSTEM: {
+					RES r=v;
+					FileSystemDock *file_system_dock=EditorNode::get_singleton()->get_scenes_dock();
+					file_system_dock->navigate_to_path(r->get_path());
+					// Ensure that the FileSystem dock is visible.
+					TabContainer* tab_container=(TabContainer*)file_system_dock->get_parent_control();
+					String file_system_title=TTR("FileSystem");
+					for (int i=0; i<tab_container->get_tab_count(); i++) {
```

Comment:
I added it.
Fun fact: For deleted resources, the context menu entry does not appear anyway, as it checks if the resources is valid.